content/en-us/scripting/security/security-tactics.md

```diff
@@ -68,20 +68,30 @@ local newPart = remoteFunction:InvokeServer(Color3.fromRGB(200, 0, 50), Vector3.
 
 if newPart then
 	print("The server created the requested part:", newPart)
+elseif newPart == false then
```

Comment:
Can this simply be an "else" or is this accounting for some subtle distinction between it resolving to either "nil" or "false"?

Reply:
In this example a simple "else" can be used, yes. The "false" return is meant to very clearly discern a failed request from the client. If the event had the chance to return "nil" (ie. retrieving some data from the server that may not exist) then for both debugging and UX purposes we'd want to be able to discern between a type checking error and a valid but empty result.

It isn't applicable to this example, so can be omitted if necessary, but I think it serves as a good general example for users. Up to you!

Reply:
Sounds good, we'll leave it as-is for clarity.